networking/network_security/configuring-ipsec-ovn.adoc

```diff
@@ -10,6 +10,20 @@ By enabling IPsec, you can encrypt both internal pod-to-pod cluster traffic betw
 
 IPsec is disabled by default. You can enable IPsec either during or after installing the cluster. For information about cluster installation, see xref:../../installing/overview/index.adoc#ocp-installation-overview[{product-title} installation overview].
 
+[NOTE]
+====
+Upgrading your cluster to {product-title} {product-version} when the `libreswan` and `NetworkManager-libreswan` packages differ between {product-title} versions leads to two consecutive compute node reboot operations. For the first reboot, the Cluster Network Operator (CNO) applies the IPsec configuration to compute nodes. For the second reboot, the Machine Config Operator (MCO) applies the latest machine configs to the cluster.
+
+Before upgrading your cluster, set the `paused` parameter to `true` in the `MachineConfigPools` custom resource (CR) that groups compute nodes as demonstrated with the following command:
```

Comment:
you could mention user can follow control-plane-only-update method as described here https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/updating_clusters/performing-a-cluster-update#control-plane-only-update.